### ISSUE TITLE
Posts doesn't show local time zone

### ISSUE THREAD

**Issue by zomidaily** [Automattic/facebook-instant-articles-wp#813]
## Steps required to reproduce the problem
I'm in Kuala Lumpur, Malaysia (GMT+8). Whenever I make a new post, it doesn't reflect the local time zone in Instant Articles' post. The time zone shows is much more ahead, and make confused my readers.

## Expected Result
My WordPress' posts are showing correctly the local time zone, so I expect the Instant Articles follows the local time zone too.

## Version Info
* Plugin version: 4.0.5
* WordPress version: 4.8.3
* PHP version: 7.0

**Comment by pestevez** [comment]
Please post your Source and Transformed markup for any of the articles with the issue. Thanks

**Comment by zomidaily** [comment]
Source Markup
https://gist.github.com/zomidaily/2d7a469422caca74cea335d05bd121d9

Transformed Markup
https://gist.github.com/zomidaily/f18860d58080fe8a7fce290aee1ac657

**Comment by pestevez** [comment]
Thanks, @zomidaily - Can you please share a screenshot of one of your posts (with the expected date and time), and one of an Instant Article with the Instant Articles where it shows the wrong date and time?

**Comment by zomidaily** [comment]
Hi @pestevez 
This is the actual link https://www.zomidaily.org/iphone-x-25-tawh-nungak-khat-propose-bawl/ and the attached shows differnet times.

![timezone](https://user-images.githubusercontent.com/4189681/32685419-d0d97c44-c6cb-11e7-9240-ea8224a46695.jpg)

**Comment by zomidaily** [comment]
Hi @pestevez 
I think I have solve the problem finally. Previously, I have set the timezone in WordPress Setting as UTC+8 since I live in that timezone. I am not aware of the city name, but today, I browse around and notice that there is Kuala Lumpur. After I set it to Kuala Lumpur, the time seems to be correctly display in Instant Article now.

![time](https://user-images.githubusercontent.com/4189681/32686687-fbbe3df0-c6e4-11e7-93c8-84f9861d3174.jpg)
